lib/lich/gameobj.rb

```diff
@@ -9,15 +7,15 @@ def method_missing(*); end
 
 # Tell GameObj where to find the GameObj data XML file
 class GameObj
-  DATA_DIR = Pathname.new('./dist').expand_path
-end
-
-def echo(*args)
-  pp *args
-end
-
-def respond(*args)
-  echo *args
+  require 'pathname'
```

Comment:
:+1: Pathname is one of my favorite modules in the stdlib. Way better than trying to remember whether a method lives in Dir/File/FileUtils or whatever.